.github/workflows/sync-issue-labels-compute.yml

```
@@ -0,0 +1,207 @@
+            console.log(`Wrote labels artifact payload for PR #${payload.pr_number}: ${payload.labels.length} labels`);
+
+      - name: Upload labels artifact
+        uses: actions/upload-artifact@bbbca2ddaa5d8feaa63e36b76fdaad77386f024f # v7.0.0
+        with:
+          name: pr-labels-${{ steps.compute.outputs.pr_number }}
+          path: labels.json
+          retention-days: 1
+
+  dispatch-add:
+    needs: compute-labels
+    if: ${{ needs.compute-labels.outputs.is_fork_pr != 'true' && secrets.GH_ACCESS_TOKEN != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)